apps/backend/src/lambdas/generateData.ts

```diff
@@ -119,28 +121,27 @@ export const handler = async (event?: any, context?: Context) => {
 
         if (fileType === FileTypes.TDI34) {
           const tdi34Details = parsed as TDI34Details[];
-          const tdi34Entities = tdi34Details.map(
-            (item) => new POSDepositEntity(item)
+          tdi34Details.map(
+            async (item: TDI34Details) =>
+              await posService.createPOSDeposit(new POSDepositEntity(item))
           );
-          // TODO: Needs to go to it's own service / repository
-          await reconService.mapPOSDeposit(tdi34Entities);
         }
 
         if (fileType === FileTypes.TDI17) {
           const tdi17Details = parsed as TDI17Details[];
-          const tdi17Entities = tdi17Details.map(
-            (item) => new CashDepositEntity(item)
+          tdi17Details.map(
+            async (item: TDI17Details) =>
+              await cashService.createCashDeposit(new CashDepositEntity(item))
           );
           // TODO: Needs to go to it's own service / repository
```

Comment:
I think we can remove the commented TOODs here

Reply:
Will do on the follow up PR if that is okay?